src/hypothesis/internal/conjecture/utils.py

```diff
@@ -68,7 +68,11 @@ def byte_distribution(random, n):
         else:
             probe = upper - v
         return int_to_bytes(probe, n)
-    probe = int_from_bytes(data.draw_bytes(nbytes, byte_distribution)) & mask
+
+    probe = gap + 1
+    while probe > gap:
```

Comment:
Because this loop always executes at least once by design, coverage is complaining about it. If you add a "# pragma: no branch" to it that will fix it.